src/mon/MonCommands.h

```diff
@@ -424,7 +424,8 @@ COMMAND("fs set " \
         "|standby_count_wanted|session_timeout|session_autoclose" \
         "|allow_standby_replay|down|joinable|min_compat_client " \
 	"name=val,type=CephString "					\
-	"name=yes_i_really_mean_it,type=CephBool,req=false",			\
+	"name=yes_i_really_mean_it,type=CephBool,req=false "		\
+	"name=yes_i_really_really_mean_it,type=CephBool,req=false",	\
```

Comment:
From a user's POV, having two `yes_i_really_mean_it` flags doesn't really make sense out of context, as it's not clear when to use the first one and when to use the other one. 

And we really need two `yes_i_really_mean_it` flags?

Reply:
Sorry for the rebase problem, so I close this pr.

Thanks for your review, but the codes you doubt are already merged to master branch.